### ISSUE TITLE
Add notes/materials for respective branch and sems

### ISSUE THREAD

**Issue by coderjolly** [CoderJolly/IPU-Engineering-Notes#2]

**Comment by MansiAyer** [comment]
I'm not sure I understand what you mean. Could you please elaborate?

**Comment by coderjolly** [comment]
This repo is for maintaining notes/materials for people studying in IP university and ppl can contribute here by adding notes/materials for different branches(CSE, IT, ECE, EEE) and for respective semesters

**Comment by MansiAyer** [comment]
Nice! I'm not from IP, but I'm an FE student. Can I still make a contribution to this with the study material I have?

**Comment by coderjolly** [comment]
Yeah, sure. As long as your subjects match with syllabus. There is a "4 year pdf" in the repo, you can consult that.

**Comment by coderjolly** [comment]
yes pls
